random_data.go

```diff
@@ -362,3 +385,75 @@ func FullDateInRange(dateRange ...string) string {
 func Timezone() string {
 	return randomFrom(jsonData.Timezones)
 }
+
+///////////// random filenames ///////////////////////////
```

Comment:
We don't really use this commenting style anywhere else, perhaps we should remove it here as well?